README.md

```diff
@@ -736,6 +746,8 @@ retryMaxInterval which is maximum time interval between two subsequent retries
             authenticator.setURL(<IAM_AUTH_URL>);
             authenticator.validate();
             testService = IbmKeyProtectApi.newInstance(authenticator, maxRetry, retryMaxInterval);
+            // To work with key ring use following
+            // testService = IbmKeyProtectApi.newInstance(authenticator, keyRingId, maxRetry, retryMaxInterval);
```

Comment:
I would leave this out, since your example is about setting max retries and interval
```suggestion
```